test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1376,19 +1372,14 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
         $result.Error.FullyQualifiedErrorId | Should Be "WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand"
     }
 
-    <#
     It "Invoke-RestMethod supports request that returns page containing UTF-8 data." {
 
-        $command = "Invoke-RestMethod -Uri http://httpbin.org/encoding/utf8 -TimeoutSec 5"
-
-        $result = ExecuteWebCommand -command $command
+        $uri = Get-WebListenerUrl -Test 'Encoding/Utf8'
+        $command = "Invoke-RestMethod -Uri '$uri'"
 
-        # Validate response content
-        # TODO: There is a bug on ConvertFrom-Json that fails for utf8.
-        $result.headers.Host | Should Match "httpbin.org"
-        $result.headers.'User-Agent' | Should Match "WindowsPowerShell"
+        $result = ExecuteWebCommand -command $command        
+        $Result.Output | Should Match '⡌⠁⠧⠑ ⠼⠁⠒  ⡍⠜⠇⠑⠹⠰⠎ ⡣⠕⠌'
```

Comment:
Is this test sufficient? Do we really check Utf8 not Utf16/32 here? 
Can we also protect from changing the encoding of this file? Does it make sense use `u{...} ?
The source file contains text in many languages. Does it make sense to ckeck multiple languages.

Reply:
I wasn't clear on what the original tests were trying to accomplish and I'm not proficient with encoding to know what is a good test or not. It was clear to me that they were definitely not sufficient for UTF8, so I tried to add something that would at least eliminate ASCII. After the response headers are available for `Invoke-RestMethod`, it would be possible to test the response encoding similar to the `Invoke-WebRequest`. But I'm definitely open to suggestion on how to improve these tests.

Reply:
On the source file. If we can narrow down a specific UTF8 character or string, We can switch to generating the string from the controller and remove the view. That way we do not need to worry about the encoding change. I'd need guidance on what to put there, though.

Reply:
I think this is an improvement over what we had before.  If you think the tests can be further improved, perhaps an issue would be a better way to address the feedback.

Reply:
I think this is good. If my research on this is correct, It's no easy task to test a string for its encoding This at least include some UTf8 chars.

Opened #4890 for further improvements

Reply:
Thanks for open tracking Issue!

Closed.